frontend/packages/ceph-storage-plugin/src/components/ocs-install/internal-mode/install-wizard-steps/review-and-create.tsx

```diff
@@ -76,11 +80,15 @@ export const ReviewAndCreate: React.FC<ReviewAndCreateProps> = ({
         <ReviewListTitle text="Configure" />
         {(encryption.clusterWide || encryption.storageClass) && (
           <>
-            <ReviewListBody>
+            <ReviewListTitle text="Configure" />
+            <ReviewListBody noValue={!kms.hasHandled}>
               <p className="ocs-install-wizard__review-encryption">Enable Encryption</p>
-              {encryption.advanced && kms.hasHandled && (
-                <p>Connected to external key management service: {kms.name}</p>
+              {encryption.advanced && (
```

Comment:
Why not `kms.hasHandled` here ?
```
              {encryption.advanced && (
```

Reply:
I have at the other place too, it's not required.